src/test/java/util/SetTest.java

```diff
@@ -0,0 +1,63 @@
+package util;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    // Test Case 구현
+    @Test
+    @DisplayName("size 결과 테스트")
+    public void size() {
+        //given
+        //when
+        int result = numbers.size();
+
+        //then
+        assertThat(result).isEqualTo(3);
```

Comment:
컬렉션을 바로 테스트할 수 있도록 지원하고 있습니다.
```suggestion
        assertThat(numbers).hasSize(3);
```